### PR TITLE
Update llvm_tot anchor and LLVM_TOT_VERSION file

### PR DESCRIPTION
**Pull request by nathanchance**
Part of #176.

### CODE REVIEW

File: generator.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 llvm_versions:
-  - &llvm_tot 13
+  - &llvm_tot 14
   - &llvm_latest 12
```
Comment (nickdesaulniers) [inline generator.yml:3]: Do you want to add anchors for `llvm_12` and bump `llvm_latest` to 13?

File: LLVM_TOT_VERSION [hunk 2]
```diff
@@ -1 +1 @@
-13
+14
```
Comment (nickdesaulniers) [inline LLVM_TOT_VERSION:1]: Doesn't running `generate.sh` update this? Has it been being modified for previous patches, just not committed?

**Review by nickdesaulniers — APPROVED** [review]

File: LLVM_TOT_VERSION (hunk 2 above)
Comment (nathanchance) [reply LLVM_TOT_VERSION:1]: Yes to both questions.

File: generator.yml (hunk 1 above)
Comment (nathanchance) [reply generator.yml:3]: Not yet, TuxSuite does not have clang-13 yet: https://gitlab.com/Linaro/tuxsuite/-/issues/129

I can change the `llvm_latest` anchor to `llvm_12` and update all of the builds accordingly with this pull request then add `llvm_latest` back as `13` when the images are available? Choice is up to you.

**Comment by nathanchance** [comment]
Looks like `clang-13` is live in TuxSuite so I will update this PR to do the rest of #176.

https://gitlab.com/Linaro/tuxsuite/-/issues/129

**Comment by nathanchance** [comment]
Already, this should close out #176.

For the `clang-13` builds, I just copied all of the builds we are doing for LLVM 14. If you want me to slim up the list, I can but it might not be a terrible idea to keep all of that coverage so we can spot ToT breakages more easily.

**Pull request merged**